test/Libraries/Microsoft.Extensions.AI.Tests/ChatCompletion/FunctionInvokingChatClientTests.cs

```diff
@@ -546,6 +546,61 @@ public async Task SupportsConsecutiveStreamingUpdatesWithFunctionCalls()
         Assert.Equal("OK bye", singleUpdateContent.Text);
     }
 
+    [Fact]
```

Comment:
It'd be good to have a few more tests validating:
- It's null when we expect it to be null.
- It does indeed propagate across async hops.